lib/ngx/balancer.md

```diff
@@ -237,6 +237,29 @@ This function was first added in the `0.1.7` version of this library.
 
 [Back to TOC](#table-of-contents)
 
+recreate_request
+------------
+**syntax:** `ok, err = balancer.recreate_request()`
+
+**context:** *balancer_by_lua&#42;*
+
+Regenerates the request buffer for sending to the upstream server. This is useful, for example
```

Comment:
Maybe better to stay consistent with wording? i.e. use "recreate" instead of "regenerate" which is the terminology used everywhere else when referring to this API.